data/Fedora.yaml

```diff
@@ -1,8 +0,0 @@
----
-systemd::accounting:
```

Comment:
my problem with this is that it actually breaks Fedora. Aren't the settings the same on all operating systems nowadays? can't we move that to init.pp instead?

Reply:
I'm heading to redo the PR with adding new OSes before dropping old. So this file will stay.